content/testnet.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Setup Rubix Node and Joining the Testnet
+aliases:
+  [
+    "/setup",
+    "/rbt",
+    "/dist",
+    "/start",
+  ]
+geekdocBreadcrumb: false
+weight: 1
+---
+
+Faucet tokens are premined tokens available for users to test and interact with the Rubix testnet.
+
+When a new node joins the testnet, it will have 0 tokens to start with. To help users get started, the Rubix testnet provides a faucet service where users can request premined test tokens. These tokens are not earned through validation but are instead distributed for testing and development purposes.
+
+To request test tokens, users must first create a node in the Rubix testnet and generate a Decentralized Identifier (DID). The DID is then submitted to the faucet, which allocates a fixed number of test tokens to the requesting node.
+
+Test tokens are free and allow users to experiment with the testnet’s functionality, such as performing token transfers, validating token chains, testing smart contracts etc. Unlike mainnet tokens, test tokens do not hold any real-world value and cannot be transferred to the main network.
+
+
+## Pre-requisites  
+
+### 1. Install Go 
+
+Any computer or personal laptop with 8GB of RAM and a decent internet connection can seamlessly run Rubix for token transfers / become validator / mining.
```

Comment:
We should we use the word "stable internet" not decent.   Above we said all tokens are premised , so what is mining in testnet?

Reply:
Right, it was already mentioned in the learn site. Took from that